LoopKit/QuantityFormatter.swift

```diff
@@ -97,16 +97,21 @@ open class QuantityFormatter {
     ///
     /// - Parameters:
     ///   - quantity: The quantity
-    ///   - unit: The value. An exception is thrown if `quantity` is not compatible with the unit.
+    ///   - unit: The unit. An exception is thrown if `quantity` is not compatible with the unit.
+    ///   - includeUnit: Whether or not to include the unit in the returned string
     /// - Returns: A localized string, or nil if `numberFormatter` is unable to format the quantity value
-    open func string(from quantity: HKQuantity, for unit: HKUnit) -> String? {
+    open func string(from quantity: HKQuantity, for unit: HKUnit, includeUnit: Bool = true) -> String? {
         let value = quantity.doubleValue(for: unit)
 
```

Comment:
I was suggesting a much simpler change. Just do this, instead of all the other code changes.

```suggestion
if !includeUnit {
    return numberFormatter.string(from: value)
}
```

Reply:
Looked right over that. Thanks!